### ISSUE TITLE
Cannot launch XMR-Stak from Awesome Miner:  "The requested operation requires elevation"

### ISSUE THREAD

**Issue by Grimm2017** [fireice-uk/xmr-stak#256]
According to the developer of Awesome Miner this is the first miner software he has seen that will not start because of required elevation.  Any idea what's going on with that?  I have tried turning off UAC, setting the .exe to run as administrator, etc, nothing works.

**Comment by psychocrypt** [comment]
Do you see the UAC dialog after the start?

**Comment by Grimm2017** [comment]
No UAC dialog.  I also verified I have the user added to the "Lock pages in memory" security policy.

**Comment by Grimm2017** [comment]
I found a fix.  If I set local group policy Computer Configuration > Windows Settings > Security Settings > Local Policies > Security Options -> User Account Control: Run all administrators in Admin Approval Mode to Disabled then Awesome Miner will launch XMR-Stak ok.  The default setting for this is Enabled.

Hope this helps.  It would be nice if Stak would launch without having to change this setting as it reduces the OS security but at least I know how to make it work now.
